content-repo/extra-docs/packs/Malware_Investigation_&_Response.md

```diff
@@ -0,0 +1,196 @@
+  
+    **Note**:  
+    Some playbook inputs can be configured on the [MDE Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/mde-malware---investigation-and-response) playbook.
+
+- #### [CrowdStrike Falcon](https://xsoar.pan.dev/marketplace/details/CrowdStrikeFalcon)
+  **Main Playbook**: [CrowdStrike Falcon Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/crowd-strike-falcon-malware---investigation-and-response)
+
+  Noteworthy feature in this pack:
+   - **Allow and Block IOCs Per Device Group**
+
+     Due to the complexity of managing multiple devices and exclusions, a policy may require certain modifications to fit each device group. This playbook enables choosing which device groups (specific or all groups) will be affected by your exclusions. In order to enable these settings use the playbooks input ApplyOCGlobally and HostGroupName which can be defined in the CrowdStrike Falcon - True Positive Incident Handling and CrowdStrike Falcon - False Positive Incident Handling playbooks.
+
+   - **Pivoting from CrowdStrike Detections to Incidents**
+
+    CrowdStrike Falcon has the concepts of detections which are single alerts and Incidents which can contain multiple detections. The CrowdStrike Falcon integration supports fetching incidents as well as detections. As the data provided for fetched CrowdStrike incidents is limited the pack contains the playbook CrowdStrike Falcon - Get Detections by Incident which enables to list CrowdStrike detections that are associated to a CrowdStrike incident and add all the relevant data from the detection to the XSOAR incident.
+ 
+
+**Notes**:  
+
+- The main playbooks contain most of the common playbook inputs across vendor playbooks, however specific vendor inputs are not in the main playbook. For example:  
+
+  - In the [CrowdStrike Falcon Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/crowd-strike-falcon-malware---investigation-and-response) playbook, you may find the following inputs:  
+  (which are responsible for managing IOCs through the EDR system)
+    - `HostGroupName`  
+    - `ApplyIOCGlobally`
+  - In the [MDE Malware - Investigation and Response](https://xsoar.pan.dev/docs/reference/playbooks/mde-malware---investigation-and-response) playbook, you may find the following inputs:   
+    (which are responsible setting up the Pro-active activities)
+    - `ActionTask` 
+    - `AutoAVScan` 
+    - `AutoAutomatedInvestigation`
+    - `AutoCollectinvestigationPackege`
+
+- XSOAR supports pre-processing rules for incidents as detailed [here](https://xsoar.pan.dev/docs/incidents/incident-pre-processing), which can be used for handling incidents that share the same incident information (e.g. alert names, IPs, hashes, etc).
+However, XSOAR also supports deduplication. Unlike Pre-processing rules, deduplication can use a configurable similarity setting and not just an exact value match.
```

Comment:
@ssokolovich 
Awesome
Maybe I would also add the input name that enables deduplication in the master playbook